src/wrapper.jl

```diff
@@ -106,22 +214,952 @@ const FT_PURGE_TX = 2
   FT_OTHER_ERROR,
   FT_DEVICE_LIST_NOT_READY)
 
-@enum(
-  FTWordLength,
-  BITS_8 = FT_BITS_8,
-  BITS_7 = FT_BITS_7)
+# Types
+# 
 
-@enum(
-  FTStopBits,
-  STOP_BITS_1 = FT_STOP_BITS_1,
-  STOP_BITS_2 = FT_STOP_BITS_2)
+# FT_GetDeviceInfoList Struct
+struct FT_DEVICE_LIST_INFO_NODE
+  flags::ULONG
+  typ::ULONG
+  id::ULONG
+  locid::DWORD
+  serialnumber::NTuple{16, Cchar}
+  description::NTuple{64, Cchar}
+  fthandle_ptr::Ptr{Cvoid}
+end
 
-@enum(
-  FTParity,
-  PARITY_NONE = FT_PARITY_NONE,
-  PARITY_ODD  = FT_PARITY_ODD,
-  PARITY_EVEN = FT_PARITY_EVEN,
-  PARITY_MARK = FT_PARITY_MARK,
-  PARITY_SPACE = FT_PARITY_SPACE)
-
-  
\ No newline at end of file
+"""
+    mutable struct FT_HANDLE<:IO
+
+Holds a handle to an FT D2XX device.
+"""
+mutable struct FT_HANDLE<:IO
```

Comment:
Whitespace <:

Reply:
resolved in 72caa5437d75e79248ff099d7f8366b73db88311